enhancements/observability/multicluster-observability-addon-interoperability.md

```diff
@@ -0,0 +1,212 @@
+observability vendors.
+
+This enhancement proposal seeks to strengthen interoperability of MCOA by unifying and 
+simplifying exporting of all MCOA telemetry data (metrics, logs, traces) 
+by exposing a unified export API and consolidating export protocols. This capability
+enables users to send data from MCOA to any observability vendor and apply 
+fine-grained filtering and routing on exported data to configurable sinks.
+
+## Motivation
+
+At the moment exporting all telemetry data from OpenShift is fragmented to tree 
+technology stacks (Prometheus, ClusterLoggingForwarder, OpenTelemetry collector). 
+Every tool uses a different configuration API, export protocol and provides (or does 
+not provide) filtering/PII capabilities.
+
+### Prior art and user requests
+
+* Red Hat OpenShift as OpenTelemetry (OTLP) native platform: https://www.redhat.com/en/blog/red-hat-openshift-opentelemetry-otlp-native-platform
+* Export in-cluster metrics to 3rd party vendor https://github.com/openshift/cluster-monitoring-operator/issues/2000
+* Exporting metrics to Dynatrace https://issues.redhat.com/browse/OBSDA-433
+* Export all metrics to Dynatrace https://issues.redhat.com/browse/OBSDA-450
+* Customer asking to export metric to Splunk https://redhat-internal.slack.com/archives/C04TFRRKUA2/p1687853284985279
+
+### User Stories
+
+* As a fleet administrator, I want to export all telemetry signals collected by MCOA to an OTLP compatible endpoint(s).
+* As a fleet administrator, I want to filter sensitive data before it is exported to MCO telemetry store or 3rd party OTLP endpoint.
+* As a fleet administrator, I want to decide which data is exported to MCO telemetry store or 3rd party OTLP endpoint.
+
+### Goals
+
+* Use a single protocol (OTLP) for exporting all data to MCO and/or 3rd party system.
+* Provide a single configuration API on MCO CRD for exporting all telemetry data.
+* Provide unified filtering and routing capabilities for all exported telemetry data.
+
+### Non-Goals
+
+* Data visualization and querying.
+
+## Proposal
+
+The following sections describe in detail the required resources as well as the workflow to enable collection and forwarding of logs and traces on an RHACM managed fleet of OCP clusters.
+
+The following section describes how data exportation, routing and filtering is 
+configured in MCO and MCOA.
+
+### Workflow Description
+
+1. Configure OTLP endpoint in MCO (`MultiClusterObservability`) CR.
+2. The MCOA configures an additional OTLP exporter in the OpenTelemetry collector. The 
+   exporter is in the pipeline that receives all data.
+3. (optional) Filtering (e.g. for PII) can be configured in 
+   `OpenTelemetryCollector` CR manged by MCOA by [transformprocessor](https://github.
+   com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor/README.md).
+4. (optional) Routing can be configured in `OpenTelemetryCollector` CR managed by MCOA by
```

Comment:
Would it make sense to add data enrichment or batching as optional steps here?

Reply:
I consider batching as internal perf optimization of the protocol and not a feature of the product. 

Enrichment can come as a separate proposal once we have formalized use-cases for it.